Build with angular does not work

When i try to use the version 10.12.4 in an angular application i get build errors during start of the application:


`Error: export 'BalCoreModule' (imported as 'BalCoreModule') was not found in '@baloise/design-system-next-componen
ts/dist'`


The Import was done like this:

`import {BalCoreModule} from '@baloise/design-system-next-components/dist';`

Additional info: IDEA suggests me to use the /dist otherwise i get errors in the IDEA. 


I also tried to do the import differently:

`import {BalCoreModule} from '@baloise/design-system-next-components-angular/dist';`

Then the following errors appears:

`./projects/pc-demo-app/src/app/app.module.ts:5:0-84 - Error: Module not found: Error: Can't resolve '@baloise/design-system-next-components-angular/dist' in 'C:\Users\pub
lic\dev\IdeaProjects\premium-calculators-shared-lib\projects\pc-demo-app\src\app'


`

--- a/packages/components-angular/package.json
+++ b/packages/components-angular/package.json
@@ -54,12 +54,12 @@
   "files": [
     "dist/"
   ],
-  "main": "dist/bundles/baloise-design-system-components-angular.umd.js",
-  "module": "dist/fesm2015/baloise-design-system-components-angular.js",
-  "es2015": "dist/fesm2015/baloise-design-system-components-angular.js",
-  "esm2015": "dist/esm2015/baloise-design-system-components-angular.js",
-  "fesm2015": "dist/fesm2015/baloise-design-system-components-angular.js",
-  "typings": "dist/baloise-design-system-components-angular.d.ts",
-  "metadata": "dist/baloise-design-system-components-angular.metadata.json",
+  "main": "dist/bundles/baloise-design-system-next-components-angular.umd.js",
+  "module": "dist/fesm2015/baloise-design-system-next-components-angular.js",
+  "es2015": "dist/fesm2015/baloise-design-system-next-components-angular.js",
+  "esm2015": "dist/esm2015/baloise-design-system-next-components-angular.js",
+  "fesm2015": "dist/fesm2015/baloise-design-system-next-components-angular.js",
+  "typings": "dist/baloise-design-system-next-components-angular.d.ts",
+  "metadata": "dist/baloise-design-system-next-components-angular.metadata.json",
   "sideEffects": false
 }

